.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: csharp
```

Comment:
I'm not sure if travis supports that, I copied this from [FAKE](https://github.com/fsharp/FAKE/blob/master/.travis.yml)